governance/docs/what_is_governance.md

```diff
@@ -0,0 +1,61 @@
+
+A community's goal in designing a governance model should be making structures of participation obvious. When your project's rules are clear, contributors can engage with confidence. Taking this approach to governance can positively impact a project's long-term viability and growth.
+
+## Governance is People
+
+Simply put, "governance" refers to the rules or customs that determine who gets to do what (or is supposed to do what), how they're supposed to do it, and when.
+
+All of project governance falls into two categories: roles, or policies and procedures. For the purpose of explanation, we'll discuss each of these separately. In practice, however, they're inseparable—two sides of the same coin.
+
+## Roles
+
+A great deal of activity hinges on roles-related governance in open source projects. Think of a role as a function someone in the project performs.  Project contributors can, and often do, have multiple roles, and project roles are often fulfilled by multiple people.
+
+As a part of governance, all projects have roles, even if they are not explicitly documented.  Documenting them will help you recruit people to those roles. A minimal set of project roles will often be something like: contributor, reviewer, maintainer.
+
+When documenting your project’s roles, ask the following questions:
+
+* What roles do (or can) project contributors play?
+* What qualifies a person to play a particular role in the project?
+* What duties, privileges, and forms of authority are associated with each role?
+* What project resources are the province or responsibility of people who perform certain roles?
+
+See the [section on Roles]() for more detail and examples of how to develop and document them for your project.
+
+## Policies and Procedures
+
+While role definitions explain how specific contributors participate in the project, there are many activities in a project that involve groups of people, including people external to the project.  Examples of this include things like a release procedure or a contribution process.  These Policies and Procedures (P&P) are what is often thought of as “governance paperwork” for projects.
+
+Some useful questions to ask when developing P&P for your project include:
+
+* How do code and documentation get accepted into the project?
+* How does code get released?
+* What rules govern communication in the project?
+* When do contributors get promoted, and how?
+* How do various decisions, such as technical strategy or project resources, get made?
+* If your project has elections, how are they conducted?
+
+There is a short list of P&P that every CNCF project is expected to have, and a slightly longer list that every project will want to have.  See the [section on Policies & Procedures]() for more detail.
```

Comment:
Is this section intentionally recursive?

Reply:
IT's not recursive, that's a blank link to a future document.